sdk/cosmos/azure-cosmos/azure/cosmos/documents.py

```diff
@@ -144,7 +144,6 @@ class IndexingMode(object):
     """
 
     Consistent = "consistent"
-    Lazy = "lazy"
```

Comment:
Could we please keep this in place?
Instead, we should add a DeprecationWarning if this value (or str equivalent) is used in each method where this value can be passed in.
Otherwise we're replacing a silent change of behaviour with a raised AttributeError which is pretty breaking.

Reply:
That makes sense. I'll rework this to be non-breaking

Reply:
Take 2 is ready for review. I believe this is now non-breaking.